mappings/net/minecraft/nbt/AbstractNumberNbt.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2514 net/minecraft/nbt/AbstractNumberTag
+CLASS net/minecraft/class_2514 net/minecraft/nbt/AbstractNumberNbt
```

Comment:
`NumberNbt`

Reply:
Ignoring the Nbt naming for now, this could have just been `NumberTag`. There's no confusion that `Number` in Java is the common root class to all numeric object types.